src/visitors/cssNamespace.js

```diff
@@ -16,11 +16,11 @@ const getCssNamespace = state => {
 
 export default (path, state) => {
   const cssNamespace = getCssNamespace(state);
-
   if (
     isStyled(path.node.tag, state) &&
     path.node.quasi.quasis[0].value.cooked &&
-    !path.node.quasi.quasis[0].value.cooked.startsWith(`\n${cssNamespace} {`)
+    !path.node.quasi.quasis[0].value.cooked.startsWith(`\n${cssNamespace} {`) &&
+    path.node.tag.property.name !== 'keyframes'
```

Comment:
Please add a sanity check here for `path.node.tag.property && ...`

Reply:
I'm sorry I don't know babel internals very well, I don't know what to check :s 
`path.node.tag` is used  just above on line 20, should I do `... && (path.node.tag.property ?
 path.node.tag.property.name !== 'keyframes' : true)`